lib/connectors/memory.js

```diff
@@ -237,6 +244,38 @@ Memory.prototype.updateOrCreate = function (model, data, options, callback) {
   });
 };
 
+Memory.prototype.findOrCreate = function(model, filter, data, callback) {
+  var self = this;
+  var nodes = self._findAll(model, filter);
+
+  if (nodes.length > 0) {
+    if (filter && filter.include) {
+      self._models[model].model.include(nodes[0], filter.include, {}, function(err, nodes) {
```

Comment:
@raymondfeng is it expected for `findOrCreate` to honour `filter.include`? Do other connectors (e.g. MongoDB or MySQL) process this filter property? If not, then I am proposing to simplify this code and ignore `filter.include`.